scripts/plotting/qbo.py

```diff
@@ -67,11 +67,16 @@ def qbo(adfobj):
     print(f"\t QBO plots will be saved here: {plot_locations[0]}")
 
     # Check redo_plot. If set to True: remove old plots, if they already exist:
-    if (not redo_plot) and plot_loc_ts.is_file() and plot_loc_amp.is_file():
+    if (not redo_plot) and plot_loc_ts.is_file():
         #Add already-existing plot to website (if enabled):
+        adfobj.debug_log(f"'{plot_loc_ts}' exists and clobber is false.")
         adfobj.add_website_data(plot_loc_ts, "QBO", None, season="QBOts", multi_case=True)
+        #Continue to next iteration:
+        return
```

Comment:
I think this will result in the QBO amplitude file being skipped as well, even if it doesn't exist.  Maybe to be safe I would return to the original version, but with a `debug_log` statement that includes both `plot_loc_ts` and `plot_loc_amp`.

Reply:
All set, thanks for the insight.